libpod/networking.go

```diff
@@ -54,13 +55,24 @@ func (r *Runtime) createNetNS(ctr *Container) (err error) {
 
 	resultStruct, err := cnitypes.GetResult(result)
 	if err != nil {
-		return errors.Wrapf(err, "error parsing result from CBI plugins")
+		return errors.Wrapf(err, "error parsing result from CNI plugins")
 	}
 
 	ctr.state.NetNS = ctrNS
 	ctr.state.IPs = resultStruct.IPs
 	ctr.state.Routes = resultStruct.Routes
 
+	// We need to temporarily use iptables to allow the container
+	// to resolve DNS until this issue is fixed upstream.
+	// https://github.com/containernetworking/plugins/pull/75
+	for _, ip := range resultStruct.IPs {
+		logrus.Info("Adding Iptables run for %s", ip.Address.IP.String())
+		iptables := []string{"iptables", "-t", "filter", "-I", "FORWARD", "-s", ip.Address.IP.String(), "!", "-o", ip.Address.IP.String(), "-j", "ACCEPT"}
```

Comment:
Since you're going to use this below too, I'd have a small function that returns []string of the arguments, but the caller can pass "-D" or "-I" which the function just stuffs into the array at index 2.

Reply:
Also this function would need to check whether the IP was an IPv6 address or not, so actually just make it take *net.IPNet for which you'd pass "ip.Address.IP" and then you'd check whether ip.To4() == nil and if so, return.

Reply:
done